tests/cpp/parser_tests.cpp

```diff
@@ -4530,7 +4529,7 @@ rule public_rule
 {
 	condition:
 		(
-			false and
+			/* comment */ false and
 			true
 		)
```

Comment:
This is one of the features of our autoformatter which has been designed not to be strict, but to be gentle. Of course the and conjunction forces a new line after and this new line forces new lines inside the parentheses sector. But we do not force any new lines after or before comments here. That is up to the user if they want to add some new lines or not.